### DISCUSSION TITLE
Multiple cursors / simultaneous editing

### DISCUSSION THREAD

**Discussion by virtuallyvlad** [standardnotes/forum#2565] [Suggestions]
Allowing the user to put cursors in multiple places at once would boost productivity and make it easier to do multiple actions simultaneously.

Looking forward, it also seems that multi-cursor support will become a more standard and expected feature of text editors.  With multi-cursor support, SN can continue to keep pace with modern editors and maintain its relevance.  Aside from enhancing SN itself, multi-cursor support can help keep folks from migrating away to more feature-filled alternatives.

This is supported by Sublime Text, Atom, VSCode, Notepad++, gedit, Emacs, and Vim.

This request includes multi-cursor selection and multi-cursor editing, but there are other sub-features that enhance multi-cursor support, like tying in shortcuts/hotkeys, multiple-selections from search, selecting the next occurrence of the selection, etc. 

This would apply for the desktop app mainly, but also for the web app if possible.  I don't believe this can be done on mobile apps as of now.

**Comment by JaspalSuri** [comment]
Hi @virtuallyvlad, thanks for the feedback! This is a great suggestion. While it's not part of our roadmap, I'm sure it's something that we would like to implement at some point in the future if possible.
